docs/reference/structure/anycloud-json.md

```diff
@@ -27,21 +27,21 @@ Anycloud deployments are configured using an `anycloud.json` file located within
 }
 ```
 
-An example of an `anycloud.json` with a staging and a production deployment configuration will look like this:
+An example of an `anycloud.json` with a staging and a production deployment configuration:
 
 ```javascript
 {
   "staging": [{
     "cloudProvider": "AWS",
     "region": "us-west-1",
     "vmType": "t2.medium",
-    "credentials": "piedpiper-aws",
+    "credentials": "stripe-aws",
   }],
   "production": [{
     "cloudProvider": "AWS",
     "region": "us-west-1",
     "vmType": "t3.xlarge",
-    "credentials": "piedpiper-aws",
+    "credentials": "stripe-aws",
```

Comment:
And here

Reply:
Yeah gitbook was acting up / I wasn't using it properly